tsconfig.json

```diff
@@ -1,14 +1,12 @@
 {
   "include": ["*.js"],
   "compilerOptions": {
-    "target": "ES2020",
-    "lib": ["ES2020"],
+    "target": "ES2022",
+    "lib": ["ES2022"],
```

Comment:
I'm open to that.
Since this isn't compiling the JS.
I'd seen that more as something for another linter like: https://www.npmjs.com/package/eslint-plugin-node to pick up.
But I'm fine down-leveling the config here as an alternative.

Reply:
updated in [f4526ef](https://github.com/remarkjs/remark-lint-no-dead-urls/pull/48/commits/f4526efc1eb7812340dd9bbd7946f20abca815ba)